### ISSUE TITLE
Undefined variable in WP core caused by permalinks-cutsomizer when previewing posts

### ISSUE THREAD

**Issue by VinzzB** [yasglobal/permalinks-customizer#26]
Hello,

There seems to be a bug with your plugin when previewing a draft page.
I used a default theme, disabled all plugins and when I re-activate premalinks-customizer the error re-appears! 
You can re-create the error as follow: Create a draft page, save it (as draft) and then press the preview button.

Here are the error (or notice) messages:

```
Notice: Undefined variable: query in /home/domain/wp-includes/class-wp.php on line 248
Notice: Undefined variable: matches in /home/domain/wp-includes/class-wp.php on line 251
```

Can you have a look?
Yours sincerely
VinzzB

**Comment by samiahmedsiddiqui** [comment]
@VinzzB Hi, How are you?

Sorry for the delayed response.

Thank you for reporting this issue. I have successfully created this on my end and will fix this out.

Sami

**Comment by VinzzB** [comment]
Hi Sami,

No problem, It is a small bug anyway.

VinzzB

**Comment by samiahmedsiddiqui** [comment]
@VinzzB Issue fixed and released with `v2.6.0`. Please check and provide your valuable feedback.

Thanks,
Sami
